e2e/testdata/fn-render/path-index-ancestor/.expected/diff.patch

```diff
@@ -0,0 +1,92 @@
+diff --git a/db/resources.yaml b/db/resources.yaml
+index 685f793..35816b7 100644
+--- a/db/resources.yaml
++++ b/db/resources.yaml
+@@ -1,5 +1,3 @@
+-# Copyright 2021 Google LLC
+-#
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+@@ -11,9 +9,19 @@
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+-apiVersion: apps/v1
++apiVersion: apps/v1 # Copyright 2021 Google LLC
```

Comment:
This is strange. Something is broken some where. either in `fn render` or in one of the functions.
@Shell32-Natsu

Reply:
Might be an issue with starlark or generators in general

See my comment here https://github.com/GoogleContainerTools/kpt/pull/1696#discussion_r608851162

Reply:
This should be an issue in kyaml.

https://github.com/GoogleContainerTools/kpt/issues/1706

We need to fix it there.